app/views/static_pages/about.html.erb

```diff
@@ -19,8 +19,16 @@
   <div class='column-inner-inner'>
     <h2 class='center maroon'>Become a supporting member</h2>
     <div class='info-text'>
-      Please fill out this form if you are interested in becoming a supporting member of the gallery.
-      Members receive these sorts of benefits and also we thank them very much...
+      Supporting members are guaranteed exhibition space at our annual December Holiday Show and are invited to exhibit at other special shows during the year.  They receive member discount rates on bus trips, special outings and workshops.  They are included in members' only events and will receive the artist's reception and show invitations, as well as the gallery e-newsletter.
+      <a href="/Membership_Application.pdf"><b>Click here</b></a> to open an Adobe PDF fill-in form or print the form below, fill in the blanks and mail to the Cooperative Gallery, ATTN: Membership Coordinator, 213 State Street, Binghamton, NY 13901.
```

Comment:
Could of things-- you reference the "form below", but the form is not embedded so better to just say "Click here to open or print the application form." (no one cares that it is Adobe PDF). Then put a line break and say "Mail the completed form to:
```
<div class='info-text center'>
  <p>Cooperative Gallery</p>
  <p>ATTN: Membership Coordinator</p>
  <p> ....... (continue here)
</div>
```